<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainw</class>
 <widget class="QDialog" name="mainw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Medicina Northern Cross - N/S Pointing System</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>780</width>
     <height>621</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_dec">
    <attribute name="title">
     <string>Pointing</string>
    </attribute>
    <widget class="QLabel" name="label_1nord_3">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>230</y>
       <width>140</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>2 S</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_1nord_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>230</y>
       <width>140</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>1 S</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2nord">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>140</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>2 N</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_1nord">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>230</y>
       <width>140</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>17</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>1 N</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>751</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Brightest Visible Radio Sources @ 408 MHz from the Medicina Station</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>410</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Declination</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>410</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Left Time</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>410</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Transit Time UTC</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>410</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="text">
      <string>Flux</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>750</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>20</y>
        <width>30</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>--&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="button_move_arb">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Go!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Move to arbitrary declination</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="arbitrary_dec">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>44.5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_2n">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>2N</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_1n">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>1N</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_1s">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>1S</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_2s">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>2S</string>
      </property>
     </widget>
    </widget>
    <zorder>label_2nord</zorder>
    <zorder>label_1nord_2</zorder>
    <zorder>label_1nord_3</zorder>
    <zorder>label_1nord</zorder>
    <zorder>label</zorder>
    <zorder>label_6</zorder>
    <zorder>label_9</zorder>
    <zorder>label_11</zorder>
    <zorder>label_12</zorder>
    <zorder>label_10</zorder>
    <zorder>frame</zorder>
   </widget>
   <widget class="QWidget" name="tab_conf">
    <attribute name="title">
     <string>Configuration</string>
    </attribute>
    <widget class="QLineEdit" name="poll_time">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Polling Server every (secs)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Antenna Server IP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="server_ip">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0.0.0.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Antenna Server Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="server_port">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>7200</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>501</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Please Select the antenna section before to MOVE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_connected">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 122, 122);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>OFFLINE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_enabled">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Click here to Enable to Move the Antenna</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 122, 122);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>DISABLED</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="comment">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>501</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Get ONLINE to read the current pointing declination!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>60</y>
     <width>130</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Click here to Enable to Move the Antenna</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(122, 122, 122);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>UNDEFINED</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
